js/stores/appStore.js

```diff
@@ -585,6 +587,21 @@ const handleAppAction = (action) => {
     case AppConstants.APP_SET_DICTIONARY:
       appState = appState.setIn(['dictionary', 'locale'], action.locale)
       break
+    case AppConstants.APP_BACKUP_KEYS:
+      ledger.backupKeys(appState, action)
```

Comment:
this should be `appState = ledger.backupKeys(appState, action)` to capture any state updates that this method might make. I know it doesn't directly make any now, but the pattern should always be the same to avoid confusion